lib/rbkit/server.rb

```diff
@@ -7,7 +7,7 @@ def initialize(pub_port, request_port)
       @request_port = request_port.to_i
       [@pub_port, @request_port].each{|port| validate_port_range(port) }
       @profiler_thread = nil
-      @profiler_stop_thread = false
+      @stop_profiler_thread = false
```

Comment:
I'm afraid I don't know how to do it. What do you want tested? the instance variable itself?